biojava-structure/src/main/java/org/biojava/nbio/structure/align/util/URLConnectionTools.java

```diff
@@ -119,72 +83,66 @@ public static HttpURLConnection openHttpURLConnection(URL url, int timeout)
 	 * @throws ConnectException
 	 *
 	 * */
-	public static HttpURLConnection openHttpURLConnection(URL url)
-	throws IOException, ConnectException {
+	public static URLConnection openURLConnection(URL url)
+			throws IOException, ConnectException {
```

Comment:
I see your point - but the Java class  is java.net.URLConnection so I think it makes more sense to keep with their naming. Also it's java.net.URL not java.net.Url